x-pack/plugins/maps/public/components/global_time_checkbox.tsx

```diff
@@ -21,13 +21,21 @@ export function GlobalTimeCheckbox({ applyGlobalTime, label, setApplyGlobalTime
 
   return (
     <EuiFormRow display="columnCompressedSwitch">
-      <EuiSwitch
-        label={label}
-        checked={applyGlobalTime}
-        onChange={onApplyGlobalTimeChange}
-        data-test-subj="mapLayerPanelApplyGlobalTimeCheckbox"
-        compressed
-      />
+      <EuiToolTip
+        position="top"
+        content={i18n.translate('xpack.maps.filterEditor.applyGlobalTimeHelp', {
+          defaultMessage:
+            'Whether the time selection from the global query bar should be applied to the layer.',
```

Comment:
How about "When enabled, time filter from global search bar is applied to this layer".